scripts/null_route_helper

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
```

Comment:
>python3 [](http://example.com/codeflow?start=15&length=7)

We need this feature on 201911 branch, so please make sure it could run on python2. Seems we could not cherry-pick directly.

Reply:
If you tested the same script with both python2/3, you can change it to 
```
#!/usr/bin/env python
```

Reply:
```#!/usr/bin/env python``` doesn't work on ```202012``` image since it will point to python2, and several packages are missing. I think the only way to solve the issue is to create another PR for ```201911``` or earlier image. It's not a clean cherry-pick.

Reply:
sonic-utilities can no longer be cherry-picked to 201911. We faced it few times and suggest to create a new PR to 201911